layer/contracts/logged_data.py

```diff
@@ -32,7 +32,9 @@ class Image:
 
     def __init__(
         self,
-        img: Union["PIL.Image.Image", Path, npt.NDArray[np.complex64], "torch.Tensor"],
+        img: Union[
+            "PIL.Image.Image", Path, "npt.NDArray[np.complex64]", "torch.Tensor"
+        ],
```

Comment:
There is another one on line 106 I think

Reply:
Yep, fixed!